### PR TITLE
Add avatars to join group invitations

### PR DESCRIPTION
**Pull request by HSalmon**
[Trello card](https://trello.com/c/YOH1LQVl/2042-5-add-inviter-s-avatar-if-set-to-email-invites)

screen shot 2016-11-04 at 12 03 25 pm

### CODE REVIEW

**Comment by gdpelican** [comment]
I like this 🍏 . Are we happy with it, @mhjb @rdbartlett ?

**Comment by HSalmon** [comment]
@gdpelican, when you say you like this, which version are you referring to?

**Comment by mhjb** [comment]
I'm fine with either, with a gentle preference for avatar above

**Comment by gdpelican** [comment]
Oh, I was under the impression that the avatar above one was implemented in
this branch and ready to go.

**Comment by HSalmon** [comment]
I'll implement that version now :)

**Comment by HSalmon** [comment]
screen shot 2016-11-10 at 12 52 22 pm

@gdpelican, @mhjb

File: app/assets/stylesheets/email.css [hunk 1]
```diff
@@ -47,22 +47,37 @@ body { max-width: 600px; font-family: 'FontAwesome',"HelveticaNeue", "Helvetica
...
 }
 
-.invita-people-mailer__context {
+.invite-people-mailer__context {
   margin-bottom: 30px;
 }
 
+.invite-people-mailer__avatar-initials {
+  margin: 0 10px 10px 0;
+  border: 1px solid #dcdcdc;
```
Comment (gdpelican) [inline app/assets/stylesheets/email.css]: Do we have $border-color here?

**Comment by gdpelican** [comment]
This looks pretty good to me

**Comment by rdbartlett** [comment]
I like the design but I think it should only show if avatar type is not initials. Which is another way of saying I think initials do a pretty bad job of identifying a user.

Also what's going on with your dummy invitation text: is it full of line-breaks or is the container element really narrow?

**Comment by HSalmon** [comment]
@rdbartlett, it's full of line-breaks. Ok, although we do display initials for thread mailer emails. Also, intercom display initials as well.

**Pull request merged**